src/vignette/http/routes.clj

```diff
@@ -115,26 +31,39 @@
                                     request)
                      request))])
 
+(def wiki-context ["/:wikia:image-type/:top-dir/:middle-dir/:original/revision/:revision"
+             :wikia wikia-regex
+             :image-type image-type-regex
+             :top-dir top-dir-regex
+             :middle-dir middle-dir-regex])
+
+(def uuid-context ["/:uuid"
+                   :uuid uuid-regex])
+
+(defmacro eval-context [ctx args routes]
+  `(context ~(eval ctx) ~args ~routes))
```

Comment:
yes, compojure expects to receive Vector of values this is the only way to achieve that vector will be 'inlined' and thus passed as a vector and not var or symbol.